analyzer/codechecker_analyzer/analyzers/clangsa/ctu_autodetection.py

```diff
@@ -137,13 +71,24 @@ def analyzer_version_info(self):
             LOG.debug('Failed to invoke command to get Clang version!')
             return False
 
-        version_info = self.parser.parse(analyzer_version)
+        version_parser = version.ClangVersionInfoParser()
+        version_info = version_parser.parse(analyzer_version)
 
         if not version_info:
             LOG.debug('Failed to parse Clang version information!')
             return False
 
         self.__analyzer_version_info = version_info
+
+    @property
+    def analyzer_version_info(self):
+        """
+        Returns the relevant parameters of the analyzer by parsing the
+        output of the analyzer binary when called with version flag.
+        """
+        if not self.__analyzer_version_info:
+            return False
```

Comment:
Why don't you use `None` to express that no analyzer version information can be found?

Reply:
I kept the False because it was used previously in the other properties too.